src/styles/cards.scss

```diff
@@ -0,0 +1,129 @@
+.card {
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  padding: 50px 40px;
+  gap: 46px 48px;
+  justify-content: center;
+
+  @media (max-width: 488px) {
+    grid-template-columns: repeat(1, 200px);
+}
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, 200px);
+  }
+
+ &__container {
+    display: flexbox;
+    box-sizing: border-box;
+    width: 200px;
+    height: 408px;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    font-family: $main-font-style;
+    transition: transform 300ms;
+
+    &:hover {
+      transform: scale(1.2);
+    }
+  }
```

Comment:
For each level of nesting, you need to add 2 spaces, so in this case, you need to add two spaces to the `&__container` because you nested it inside the `card`. For `&:hover` you need to add 4 spaces or 2 spaces from the parent position because you nested it inside `container` inside `card`. It will improve readability. Please fix this in all scss files

like in this [rule](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)
![image](https://github.com/mate-academy/layout_catalog/assets/97075591/b19a8a9b-6d74-4dff-aa5c-4dd67a9e8dd6)